### PR TITLE
Issue-1690: Fix strongbox-web-ui installation in Windows

### LINKED ISSUE #1690: Fix strongbox-web-ui installation under windows
# Task Description

The [strongbox/strongbox-web-ui](https://github.com/strongbox/strongbox-web-ui) project has a `patch-antlr4` script which currently only works on Linux/Unix because of the replace path style. We need to improve this so that it works on Windows as well.

# Tasks

The following tasks will need to be carried out:

* [x] Create a `tools` folder in the `strongbox-web-ui` root path.
* [x] Move `update-webdriver.js` into `tools` and change all references in the `package.json
* [x] Create a `./tools/antlr4-patch.js` javascript which is using [`replace-in-file`](https://www.npmjs.com/package/replace-in-file) to replace the occurrences of `var fs.*` to `var fs = false;` in the files `node_modules/antlr4/FileStream.js` and `node_modules/antlr4/CharStreams.js`
* [x] Replace the current `patch-antlr4` script in the `package.json` so it runs `node ./tools/antlr4-patch.js`

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @steve-todorov

### PR DESCRIPTION
**Pull request by aosousa**
Fixes strongbox/strongbox#1690

### CODE REVIEW

File: tools/test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+bar
```
Comment (steve-todorov) [inline tools/test.txt]: Could you please remove this file? It's unnecessary :)

File: tools/antlr4-patch.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const replace = require('replace-in-file');
+const options = {
+    files: ['./node_modules/antlr4/CharStreams.js', './node_modules/antlr4/FileStream.js'],
+    from: /var fs.*/g,
+    to: 'var fs = false'
+}
```
Comment (steve-todorov) [inline tools/antlr4-patch.js]: Missing `;` at the end :)
```suggestion
};
```

**Review by steve-todorov — APPROVED** [review]
Thanks for fixing this and confirming it works on Windows now! There are two mainly cosmetic changes needed, but otherwise LGTM!  :)

File: tools/test.txt (hunk 1 above)
Comment (aosousa) [reply tools/test.txt]: Completely forgot to remove it during testing, my bad! Will do so

**Comment by steve-todorov** [comment]
Thanks for your contribution @aosousa! Congrats on your first contribution! :)

**Pull request merged**